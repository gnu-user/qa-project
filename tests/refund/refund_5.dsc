TITLE
    Refund Seller Input (Invalid Input, seller does not exist)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the refund 
    transaction. When prompted, enter a buyer which exists. When prompted, 
    enter a seller which does not exist. Verify that the seller is not 
    accepted, and an error occurs.
DEPENDENCIES
    refund_4
