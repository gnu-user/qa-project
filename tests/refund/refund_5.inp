login
administrator
refund
buyer
unknownuser