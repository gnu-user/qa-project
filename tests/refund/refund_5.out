Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the buyer's username: 
Enter the seller's username: 
Invalid username, username does not exist.